### DISCUSSION TITLE
Screencast 'Advanced search' / query string

### DISCUSSION THREAD

**Discussion by mediaconform** [livewire/livewire#4069] [1. Help] [answered]
Hi folks,

I've built a page with advanced search fields according to episode 'Advanced Search' of the livewire screencast. It works like a charm, when I'm on the page.

Now I would like to call this page via its URL while passing query parameters so it will open with prefilled search fields/values, showing the already filtered content accordingly.

From what I understand, I need to get the $filters array to take over the values from my query string, if that makes sense. 

I read in the documentation about the query string feature, but i can't get it to do the job. This is probably a very simple one, but I don't know how to do it. So any help would be greatly appreciated.

Thanks.

**Comment by joshhanley** [comment]
@mediaconform please share your code of what you have attempted so we can see if there are any issues with it 🙂

Thanks!

**Comment by mediaconform** [comment, marked as answer]
Sorry, for being so unspecific, but meanwhile I found the solution myself. It was just a matter of passing some URL query prameters to a nested livewire component and receiving these parameters by the component. Somehow I had a mental block on how to do that. Reading the livewire documentation [https://laravel-livewire.com/docs/2.x/rendering-components](https://laravel-livewire.com/docs/2.x/rendering-components) helped ... thanks for looking into it anyways.

**Reply by joshhanley** [reply]
@mediaconform no worries! Glad you got it sorted 🙂
